.gitignore

```diff
@@ -35,3 +35,6 @@ src/**/Manifest.toml
 
 # Julia System Images
 *.so
+
+*.sh
+*.out
```

Comment:
```suggestion
```
I think `.gitignore` should contain only generated files.

Reply:
Sounds good! I guess it makes life easier for people who use bash scripts to run the CM, but it's not a biggie either way